### DISCUSSION TITLE
Trekantsberegner

### DISCUSSION THREAD

**Discussion by kimvul** [Eduap-com/WordMat#102] [Support]
Hej 
Når jeg åbner Trekant, kommer billedet med trekantsløser frem, men den viser ikke nogen trekant, nogen der ved hvad jeg skal gøre?
Det er på en Mac som kører Monterey, jeg bruger GeoGebra som CAS, da den regner helt forkert hvis jeg bruger Maxima.

**Comment by Eduap-com** [comment]
Trekantsløseren er ikke forbundet til Maxima eller GeoGebra.
Intel eller Apple silicon?
Screen shots?
